go/private/sdk.bzl

```diff
@@ -149,6 +149,7 @@ go_download_sdk_rule = repository_rule(
         "_sdk_build_file": attr.label(
             default = Label("//go/private:BUILD.sdk.bazel"),
         ),
+        "patches": attr.label_list(),
```

Comment:
If we add `patches`, we should also add `patch_prefix`. Could you add that and move these attributes above the implicit `_sdk_build_file` attribute?

Reply:
I don't understand the meaning of `patch_prefix`. Could you explain it?

Reply:
Sorry about that, the attribute I suggested doesn't exist. What I really meant is `patch_args` (see the same attribute on `http_archive`), which can be used to set the length of the segment that should be stripped from patch paths via `-p`.

Reply:
I have moved attributes to `_sdk_build_file`.

Reply:
all done @fmeum